src/main/java/org/amahi/anywhere/adapter/RecentFilesAdapter.java

```diff
@@ -129,7 +142,8 @@ public void tearDownCallbacks() {
 
     @Override
     public int getItemCount() {
-        return recentFiles.size();
+        int SHIMMER_ITEM_NUMBER = 10;
```

Comment:
What if your screen is big enough and more than 10 items are visible at once?

Reply:
There can be 2 ways to solve this issue :

1. Just increase the number 10 to say, 20 (this should be fine for most practical cases)
2. Implement some way to calculate the number of items to display based on screen height at runtime.

Reply:
I guess the first one should be fine? In case there are extra items present, then that would just go to a scroll list. In the second option, it can be a potential pain point since android devices come in a variety of sizes.

Reply:
Thanks, I am implementing the first one.